lib/api/objectGetACL.js

```diff
@@ -154,7 +165,7 @@ export default function objectGetACL(authInfo, request, log, callback) {
                 if (err) {
                     log.trace('error processing request',
                         { method: 'objectGetACL', error: err });
-                    return callback(err, null, corsHeaders);
+                    return callback(err, bucket);
```

Comment:
good catch, I think I just missed changing the callback name to next (since it's an error case, we just return the error and the bucket for cors headers)
I will update when I get to addressing comments